### DISCUSSION TITLE
WPF图形绘制

### DISCUSSION THREAD

**Discussion by liucvi** [BYJRK/DotNet-Discussions#52] [WPF 开发]
请问下有没有什么好用的替代Bitmap的图形绘制的库可以在WPF里用。想要达到的效果如下：
![image](https://user-images.githubusercontent.com/28768957/224619554-03452498-8c28-478f-a24c-982bc731d9d2.png)
![image](https://user-images.githubusercontent.com/28768957/224619638-86924c6a-53b9-4a69-882e-008f3374463d.png)
谢谢十月老师：D

**Comment by BYJRK** [comment]
绘制图表可以考虑使用 [LiveCharts](https://lvcharts.com/)

下面这个图是 3D 会旋转的吗？

**Reply by liucvi** [reply]
是的 是可以3D旋转的。但是我看不出官方（Autodesk）是用什么技术做的

**Reply by BYJRK** [reply]
我没用过 Autodesk。它是基于 .NET 实现的吗？会不会是用的 js + Electron？

**Reply by LiuChao19940201** [reply]
https://github.com/helix-toolkit/helix-toolkit
这个3D可以吗？

**Comment by eijjxx** [comment]
wpf并没有太好的免费chart库。
收费的我用不起。。。

另：原来用LiveChart的时候有内存泄漏问题，之后没细研究了

**Reply by LiuChao19940201** [reply]
https://scottplot.net/
